src/middlewared/middlewared/etc_files/cni/net.d/multus.d/multus.kubeconfig.mako

```diff
@@ -0,0 +1,24 @@
+<%
+    config = middleware.call_sync('datastore.query', 'services.kubernetes', [], {'get': True})
```

Comment:
Why using the datastore? Isnt there a service interface for this datastore?

Reply:
We use it because with the service interface, we pop out the cni config as it's of no use to the user and is just used for internal handling of the CNI plugins.